<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEditRight"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File Left</string>
    </property>
    <addaction name="actionOpenLeft"/>
    <addaction name="actionSaveLeft"/>
    <addaction name="actionSaveAsLeft"/>
   </widget>
   <widget class="QMenu" name="menuFile_Right">
    <property name="title">
     <string>File Right</string>
    </property>
    <addaction name="actionSaveRight"/>
    <addaction name="actionOpenRight"/>
    <addaction name="actionSaveAsRight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFile_Right"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveLeft">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpenLeft">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveAsLeft">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSaveRight">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpenRight">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveAsRight">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
